packages/atlas/.storybook/main.ts

```diff
@@ -21,6 +21,7 @@ const config: CustomizedStorybookConfig = {
     }
     storybookConfig.build.target = ''
     const viteConfig = _viteConfig as UserConfig
+    // @ts-ignore complains about flat
```

Comment:
oh, really? I don't see any error

Reply:
My editor is complaining:
![Screenshot 2022-06-01 at 11 03 53](https://user-images.githubusercontent.com/12646744/171368537-6bfb8548-1511-4da4-8bd8-5e79d3c15899.jpg)